src/V4Router.sol

```diff
@@ -63,12 +65,11 @@ abstract contract V4Router is IV4Router, BaseActionsRouter, DeltaResolver {
     }
 
     function _swapExactInputSingle(IV4Router.ExactInputSingleParams memory params) private {
+        uint128 amountIn = (params.amountIn == OPEN_DELTA)
```

Comment:
can this be moved to a library? this is for sure shared logic / will be shared logic